packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py

```diff
@@ -516,6 +516,7 @@ def async_act(self) -> Generator:
                 prediction_response.confidence if prediction_response else None,
                 bet_amount,
                 next_mock_data_row,
+                decision_received_timestamp=self.synced_timestamp,
```

Comment:
:warning: Here you disable the NONE event!! It cannot be triggered anymore!

Reply:
Should be fixed now.